Sources/Support/PaywallExtensions.swift

```diff
@@ -87,7 +87,7 @@ extension SubscriptionStoreView {
 
 #if os(xrOS)
 @available(iOS 17.0, macOS 14.0, tvOS 17.0, watchOS 10.0, *)
-extension SubscriptionStoreView where Content == DefaultSubscriptionStoreMarketingContent {
+extension SubscriptionStoreView where Content == AutomaticSubscriptionStoreMarketingContent {
```

Comment:
This is now in sync with the `iOS` version below, so I'm going to remove the `#if`.